stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -318,6 +318,108 @@ private class TraceContext {
     internalConsistencyCheck(returnValueIdx == outputReturnValueCount)
     return traceGraphOutputs
   }
+
+  /// Bind data to the inputs of the graph and return the specialized graph.
```

Comment:
It's returning the function name, not a graph as far as i understand. What if the function is called twice? How will the two function names differ?

Reply:
Yes, it will create a new function with a new name.